### PR TITLE
Kaichao update UG

### PR DESCRIPTION
**Pull request by kaichaoang**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -230,15 +235,55 @@ Deletes the stock(s) using the stock's serial number from the inventory. Multipl
 <h4>Format</h4>
 
 ```
-delete sn/<serial number>...
+delete sn/<serial number> [sn/<serial number>]...
```
Comment (AudreyFelicio) [inline docs/UserGuide.md:238]: I think `sn/<serial number>...` is already correct.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -662,11 +735,11 @@ Warenager will suggest:
...
 
-### Generates a csv file that contains all stocks: `print`
-Generates a csv file that contains all stocks. Csv file will be named according to the user input, and the file name
+### Generates a CSV file that contains all stocks: `print`
+Generates a CSV file that contains all stocks. Csv file will be named according to the user input, and the file name
 can only contain alphanumeric characters. Users may want to sort the stocks using `sort` command
-to sort the stock in their preferred order before converting it into the csv file. The csv file is saved
-to `[root directory]/data/userInput.csv` after successfully executing the command.
+to sort the stock in their preferred order before converting it into the CSV file. The CSV file is saved
+to `[root directory]/data/userInput.CSV` after successfully executing the command.
```
Comment (AudreyFelicio) [inline docs/UserGuide.md]: I think the `CSV` on `userInput.CSV` should be lowercased as in this context it is a file extension. Same issue with other `CSV` below.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -685,25 +758,63 @@ folder circled in red.
...
+
+```
+tab
+```
+
 ### Saving data
-Data (all stocks in inventory in json) is automatically saved to
-`[root directory]/data/stockbook.json` when any of these commands is executed:
+Data (all stocks in inventory in JSON) is automatically saved to
+`[root directory]/data/stockbook.JSON` when any of these commands are executed:
```
Comment (AudreyFelicio) [inline docs/UserGuide.md]: I think the `JSON` on `stockbook.json` should be lowercased as in this context it is a file extension. Same issue with other `JSON` below.

**Review by AudreyFelicio — CHANGES_REQUESTED** [review]
Overall LGTM! Only some minor notation issues.

File: docs/UserGuide.md (hunk 3 above)
Comment (kaichaoang) [reply docs/UserGuide.md]: Fixed!

File: docs/UserGuide.md (hunk 2 above)
Comment (kaichaoang) [reply docs/UserGuide.md]: Fixed!

File: docs/UserGuide.md (hunk 1 above)
Comment (kaichaoang) [reply docs/UserGuide.md:238]: I think its better to tell user that ... refers to zero or more sn/ fields. They might think ... should mean anything else at one glance.

**Review by AudreyFelicio — APPROVED** [review]
LGTM!

**Pull request merged**